Tyler/Challenge 1/26/style.css

```diff
@@ -0,0 +1,27 @@
+
+
+body{
+  background-color: rgb(59, 59, 59);
+}
+
+.trailer-button{
```

Comment:
This styling class is missing a :hover psuedo style. There is no hovering styling.

- [x] button looks like it does on website
- [ ] hovering over button has same styling change as website.

Reply:
Added the on hover.